python/pyspark/worker.py

```diff
@@ -177,8 +177,13 @@ def process():
             process()
     except Exception:
         try:
+            exc_info = traceback.format_exc()
+            try:
+                exc_info = exc_info.encode("utf-8")
```

Comment:
In Python 2, traceback.format_exc() is already str, don't need to encode. So can I change like this:
<pre>
<code>
    exc_info = traceback.format_exc()
    if sys.version > '3':
        exc_info = exc_info.encode("utf-8")
    write_int(SpecialLengths.PYTHON_EXCEPTION_THROWN, outfile)
    write_with_length(exc_info, outfile)
</code>
</pre>